packages/loot-design/src/components/budget/rollover/TransferTooltip.js

```diff
@@ -83,11 +83,7 @@ export default function TransferTooltip({
         onUpdate={id => {}}
         onSelect={id => setCategory(id)}
         inputProps={{
-          onKeyDown: e => {
-            if (e.keyCode === 13) {
-              submit();
-            }
-          }
+          onEnter: () => submit()
         }}
```

Comment:
Small nit: can you condense this down to a one-liner?

```suggestion
        inputProps={{ onEnter: submit }}
```

Reply:
Done!